<ui version="4.0" >
 <class>LagrangianOutputForm</class>
 <widget class="QWidget" name="LagrangianOutputForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QGroupBox" name="groupBoxListing" >
     <property name="title" >
      <string>Output listing</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" >
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="comboBoxNTLAL" >
          <property name="minimumSize" >
           <size>
            <width>250</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Code_Saturne key word: NTLAL</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLineEdit" name="lineEditNTLAL" >
          <property name="minimumSize" >
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>60</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="toolTip" >
           <string>Code_Saturne key word: NTLAL</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMode" >
     <property name="title" >
      <string>Post-processing for particles</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="labelIENSI1" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IENSI1</string>
            </property>
            <property name="text" >
             <string>Trajectory mode</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QCheckBox" name="checkBoxIENSI1" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IENSI1</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="labelIENSI2" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IENSI2</string>
            </property>
            <property name="text" >
             <string>Displacement mode</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QCheckBox" name="checkBoxIENSI2" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IENSI2</string>
            </property>
            <property name="text" >
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="labelNVISLA" >
            <property name="toolTip" >
             <string>Code_Saturne key word: NVISLA</string>
            </property>
            <property name="text" >
             <string>Output frequency</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="lineEditNVISLA" >
            <property name="maximumSize" >
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Code_Saturne key word: NVISLA</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" >
          <item>
           <widget class="QLabel" name="labelNBVIS" >
            <property name="toolTip" >
             <string>Code_Saturne key word: NBVIS</string>
            </property>
            <property name="text" >
             <string>Number of particles for post-processing</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditNBVIS" >
            <property name="maximumSize" >
             <size>
              <width>80</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="toolTip" >
             <string>Code_Saturne key word: NBVIS</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>17</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>13</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="labelFormat_2" >
            <property name="text" >
             <string>Format</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QComboBox" name="comboBoxFormat" >
            <property name="minimumSize" >
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="labelOptions" >
            <property name="text" >
             <string>Options </string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QComboBox" name="comboBoxOptions" >
            <property name="minimumSize" >
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>13</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxVariables" >
        <property name="title" >
         <string>Variables selection</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" >
         <item row="0" column="1" >
          <widget class="QLabel" name="labelIVISV2" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISV2</string>
           </property>
           <property name="text" >
            <string>Particle velocity</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISV2" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISV2</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLabel" name="labelIVISV1" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISV1</string>
           </property>
           <property name="text" >
            <string>Fluid velocity seen by particles</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISV1" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISV1</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="2" column="1" >
          <widget class="QLabel" name="labelIVISTP" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISTP</string>
           </property>
           <property name="text" >
            <string>Residence time</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISTP" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISTP</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="3" column="1" >
          <widget class="QLabel" name="labelIVISDM" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISDM</string>
           </property>
           <property name="text" >
            <string>Particle diameter</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISDM" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISDM</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="4" column="0" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>16</width>
             <height>21</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="4" column="1" >
          <widget class="QLabel" name="labelIVISTE" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISTE</string>
           </property>
           <property name="text" >
            <string>Particle temperature</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISTE" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISTE</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item rowspan="2" row="4" column="3" >
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>21</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="5" column="1" >
          <widget class="QLabel" name="labelIVISMP" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISMP</string>
           </property>
           <property name="text" >
            <string>Particle mass</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISMP" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISMP</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="6" column="1" >
          <widget class="QLabel" name="labelIVISHP" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISHP</string>
           </property>
           <property name="text" >
            <string>Temperature of the coal particles</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISHP" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISHP</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="7" column="1" >
          <widget class="QLabel" name="labelIVISDK" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISDK</string>
           </property>
           <property name="text" >
            <string>Shrinking core diameter of the coal particles</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISDK" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISDK</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="8" column="1" >
          <widget class="QLabel" name="labelIVISCH" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISCH</string>
           </property>
           <property name="text" >
            <string>Mass of reactive coal of the coal particles</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISCH" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISCH</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
         <item row="9" column="1" >
          <widget class="QLabel" name="labelIVISCK" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISCK</string>
           </property>
           <property name="text" >
            <string>Mass of char of the coal particles</string>
           </property>
          </widget>
         </item>
         <item row="9" column="2" >
          <widget class="QCheckBox" name="checkBoxIVISCK" >
           <property name="toolTip" >
            <string>Code_Saturne key word: IVISCK</string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>17</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
